gobcore/model/gobmodel.json

```diff
@@ -2205,7 +2208,8 @@
             }
           },
           "geboortedatum": {
-            "type": "GOB.String",
+            "type": "GOB.SecureString",
```

Comment:
This should be a SecureDate

Reply:
SecureData does not exist. First SecureDate should be created. There's a new ticket (1300) for that.